src/database/migrations/20200516230350-create-chat-table.js

```diff
@@ -0,0 +1,46 @@
+
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Chats', {
+    uuid: {
+      allowNull: false,
+      primaryKey: true,
+      type: Sequelize.UUID,
+      defaultValue: Sequelize.UUIDV4
+    },
+    user_uuid: {
+      allowNull: false,
+      type: Sequelize.UUID
+    },
+    connection_uuid: {
+      allowNull: false,
+      type: Sequelize.UUID
+    },
+    parent_uuid: {
```

Comment:
parent uuid? what does this identify?

Reply:
That's the sender id for that particular message

Reply:
Lets's say a message is a response to another message, then the parent_uuid is the uuid  of the former message that this present message is a response to.